pkg/apis/admissionregistration/validation/validation_test.go

```diff
@@ -483,94 +485,189 @@ func TestValidateExternalAdmissionHookConfiguration(t *testing.T) {
 			expectedError: `externalAdmissionHooks[0].failurePolicy: Unsupported value: "other": supported values: "Fail", "Ignore"`,
 		},
 		{
-			name: "URLPath must start with slash",
+			name: "both service and URL missing",
+			config: getExternalAdmissionHookConfiguration(
+				[]admissionregistration.ExternalAdmissionHook{
+					{
+						Name:         "webhook.k8s.io",
+						ClientConfig: admissionregistration.AdmissionHookClientConfig{},
+					},
+				}),
+			expectedError: `exactly one of`,
```

Comment:
get slightly more specific so you include the fields too.

Reply:
I intended to copy/paste from the failing test to put in the thing to check, but then the test unexpectedly passed and I didn't...

Reply:
I include each field in at least one of the test cases now, I don't like making change-detector tests so I don't think it needs to be universal.